src/high-level/lapack-macros.lisp

```diff
@@ -0,0 +1,510 @@
+;;;; lapack-macros.lisp
```

Comment:
```suggestion
;;;; a-lisp-programmer-casually-writing-c++-templates.lisp
```

jk

Reply:
Renamed to `lapack-templates.lisp` reflect this